rpc/handlers.go

```diff
@@ -43,6 +47,143 @@ func (s *rpcServer) PostV1ClientBroadcastTxSync(ctx echo.Context) error {
 	return nil
 }
 
+// DISCUSSION: This may need to be changed when the GetSession function is actually implemented
+func (s *rpcServer) PostV1ClientDispatch(ctx echo.Context) error {
```

Comment:
Can you rename this `GetSession` (ditto elsewhere). `Dispatch` is a very legacy term.